### ISSUE TITLE
PHP 8 gives an error on front-end forms

### ISSUE THREAD

**Issue by GerritChan** [joppuyo/acf-image-aspect-ratio-crop#47]
Using acf_form on the front-end, i'm getting the following error: 

```
acf_form(array(
      'post_id' => 'user_' . get_current_user_id(),
      'field_groups' => array('group_5fcdeb724f2b2'),
      'uploader' => 'basic',
      'submit_value'	=> 'Edit'
 ));
```

Fatal error: Uncaught TypeError: implode(): Argument #2 ($array) must be of type ?array, string given in C:\Bitnami\wampstack-8.0.0-1\apache2\htdocs\plugins\acf-image-aspect-ratio-crop\fields\class-npx-acf-field-image-aspect-ratio-crop-v5.php:496

**Comment by joppuyo** [comment]
Hey, I'll take a look at fixing this ASAP. Seems to be a simple fix, the order of parameters needs to be flipped, the legacy order worked all the way up to PHP 7.4 but it was removed in 8.0.

**Comment by GerritChan** [comment]
Turned out that switching the implode() parameters fixes the problem! 

in class-npx-acf-field-image-aspect-ratio-crop-v5.php

```
/* PHP 8 error */
<?php echo implode($mime_array, ','); ?>

/* PHP 8 compatible */
<?php echo implode(',', $mime_array); ?>
```

**Comment by joppuyo** [comment]
It took a bit longer than I expected to get around going this but this issue has been fixed in v5.0.6.
